browser/extensions/brave_component_loader.cc

```diff
@@ -112,7 +114,10 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
   // it would cause a bug with loading brave://wallet not loading
   // if a tab is left open and you restart the browser.  That would
   // need to be fixed first.
-  if (ExtensionPrefs::Get(profile_)->
+  // Does not load if project id is not configured.
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
+  bool has_project_id = env->HasVar("BRAVE_INFURA_PROJECT_ID");
```

Comment:
nit: is there some kind of util we can put this string or logic in to? It's repeated at least 3 times. e.g. `environmentHasInfuraProject()`

Reply:
Well I could set up a helper util and replace two instances, but in the unit test I need access to env to set the var if it doesn’t exist

Reply:
a util has been created for this.